app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,4 @@
 // Specific CSS for your home-page
+body {
+  background-color: #a00;
```

Comment:
@asamariya , you think it should be #f00?

Reply:
much better